README.md

```diff
@@ -332,11 +150,11 @@ hosts which form a tree. Every protocol defines the steps needed to
 accomplish the calculations, and the framework makes sure that all 
 messages are passed between the hosts.
   
-The directory-structure is as follows:
-
-* [`lib/`](lib/): contains all internally used libraries
-* [`lib/sda/`](lib/sda/): basic definition of our framework
-* [`protocols/`](protocols/): one directory per protocol, holds both the 
-definition and eventual initialisation necessary for a simulation
-* [`simul/`](simul/): used for running simulations on localhost and 
-DeterLab
+## Directory-structure
+
+* [`sda/`](sda/): basic definition of our framework
```

Comment:
the framework which is used to handle the messages, the overlay tree layer, the protocols and the services
??